### PR TITLE
Simplified Chinese Localization

### PR DESCRIPTION
**Pull request by Samekichi**
Truly love this mod so make a localization :D And also some notices: 

- I've added some color format codes which you may delete if unlike them. 
- The lang template is missing upgrade instruction and upgrade category strings of the bows' tooltip.

### CODE REVIEW

File: src/main/resources/assets/extrabows/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  "freeze_upgrade.description.text": "雪会覆盖箭矢落点附近的固体方块, 水则凝成浮冰, 还有几率生成雪傀儡; 被射中的生物会被减速",
+
+  "explosive_upgrade.description.text": "箭矢的落点处会产生一次爆炸",
+
+  "double_upgrade.description.text": "每次消耗并射出 2 支箭矢",
+
+  "triple_upgrade.description.text": "每次消耗并射出 3 支箭矢",
+
+  "push_upgrade.description.text":"将箭矢落点半径 5 格内的动物推远",
```
Comment (EldersJavas) [inline src/main/resources/assets/extrabows/lang/zh_cn.json:63]:
```suggestion
  "push_upgrade.description.text":"将箭矢落点半径 5 格内的生物推远",
```

File: src/main/resources/assets/extrabows/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  "flying_upgrade.description.text": "你会御箭飞行 §7(按 shift 可以脱离, 从而可以替代烟花起飞)",
+
+  "bone_meal_upgrade.description.text": "箭矢落点半径 3 格内分别有 50% 的几率生成植物",
```
Comment (EldersJavas) [inline src/main/resources/assets/extrabows/lang/zh_cn.json]:
```suggestion
  "bone_meal_upgrade.description.text": "箭矢落点半径 3 格内有 50% 几率生成植物",
```

File: src/main/resources/assets/extrabows/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+  "explosive_upgrade.description.text": "箭矢的落点处会产生一次爆炸",
+
+  "double_upgrade.description.text": "每次消耗并射出 2 支箭矢",
+
+  "triple_upgrade.description.text": "每次消耗并射出 3 支箭矢",
+
+  "push_upgrade.description.text":"将箭矢落点半径 5 格内的动物推远",
+
+  "flying_upgrade.description.text": "你会御箭飞行 §7(按 shift 可以脱离, 从而可以替代烟花起飞)",
```
Comment (EldersJavas) [inline src/main/resources/assets/extrabows/lang/zh_cn.json]:
```suggestion
  "flying_upgrade.description.text": "你会御箭飞行 §7(按 shift 脱离, 从而可以替代烟花起飞)",
```

**Review by EldersJavas — APPROVED** [review]
本想翻译的,结果翻译一半偷看下pr发现已经有大佬翻译了...
话说两个月了作者还没mange吗?

有几点修改的建议,勿喷:)

File: src/main/resources/assets/extrabows/lang/zh_cn.json (hunk 1 above)
Comment (Samekichi) [reply src/main/resources/assets/extrabows/lang/zh_cn.json:63]: 确实年代久远了233
当初写成`动物`似乎是因为只对动物生效，但我不记得这个进阶模板在 1.15.2 里测试的结果了。所以如果是生物都能被击退，就告诉我一哈

**Comment by MrMarnic** [comment]
Has been added to github. Thank you.

**Pull request closed without merging**